superset-frontend/src/components/Checkbox.jsx

```diff
@@ -32,15 +32,17 @@ export default function Checkbox({ checked, onChange, style }) {
         role="button"
         tabIndex={0}
         className={`fa fa-check ${
-          checked ? 'text-primary' : 'text-transparent'
+          checked ? 'text-white' : 'text-transparent'
         }`}
         onClick={() => {
           onChange(!checked);
         }}
         style={{
-          border: '1px solid #aaa',
+          border: `${checked ? '1px solid #20A7C9' : '1px solid #aaa'}`,
           borderRadius: '2px',
           cursor: 'pointer',
+          boxShaddow: 'inset 1px 0px 0px #E0E0E0',
+          backgroundColor: `${checked ? '#20A7C9' : 'transparent'}` 
```

Comment:
I didn't realize this component exists. Thoughts on deprecating this in favor of https://github.com/preset-io/incubator-superset/blob/master/superset-frontend/src/components/IndeterminateCheckbox.tsx which is using the new Icon component and emotion styles?

Reply:
sounds like a good idea to me. @rusackas any thoughts on this?

Reply:
I'm all for consolidation! Use our Checkbox wherever checkboxes are used!

Reply:
Also let's never use hardcoded colors!